nixos/modules/services/audio/mpd.nix

```
@@ -1,4 +1,7 @@
 { config, lib, pkgs, ... }:
+
+with lib;
```

```suggestion
inherit (lib) isList isAttrs isBool;
```
#208242
Probably you need to inherit even more stuff from lib.

---

Updated